src/Forte.Optimizely.ContentUsage/Api/Services/ContentUsageService.cs

```diff
@@ -27,7 +27,7 @@ public class ContentUsageService
         _publishedStateAssessor = publishedStateAssessor;
     }
 
-    public IEnumerable<string> GetPageUrls(ContentUsage contentUsage)
+    public IEnumerable<string> GetPageUrls(EPiServer.DataAbstraction.ContentUsage contentUsage)
```

Comment:
For now I decided to attach full namespace for ContentUsage from EPiServer.DataAbstraction namespace.
@darekhalecki Do you have any suggestion to do it in different way?

Reply:
We can set an alias like this (if this looks cleaner for you):
`using ContentTypeUsage = EPiServer.DataAbstraction.ContentUsage;`

Reply:
I though about that, but it overrides existing class name for this file. I try to give it a "EPiServerContentUsage" name.